### PR TITLE
Login and registration form

### PR DESCRIPTION
**Pull request by BNikolov13**

### CODE REVIEW

File: Frontend/public/favicon.ico
Comment (DameStoilovski99) [inline Frontend/public/favicon.ico:1]: delete this file

File: Frontend/public/logo192.png
Comment (DameStoilovski99) [inline Frontend/public/logo192.png:1]: delete this file

File: Frontend/public/favicon.ico
Comment (BNikolov13) [reply Frontend/public/favicon.ico:1]: okey

File: Frontend/public/logo192.png
Comment (BNikolov13) [reply Frontend/public/logo192.png:1]: okey

**Pull request merged**